From foo@baz Wed Aug 19 16:21:28 PDT 2009
Date: Wed, 19 Aug 2009 16:21:28 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: coding style cleanup of include/HvVpApi.h

Coding style fixes for include/HvVpApi.h

All of the include/Hv*.h files should be merged eventually...

Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/include/HvSynicApi.h |    4 +-
 drivers/staging/hv/include/HvVpApi.h    |   48 ++++++++++++++------------------
 2 files changed, 23 insertions(+), 29 deletions(-)

--- a/drivers/staging/hv/include/HvSynicApi.h
+++ b/drivers/staging/hv/include/HvSynicApi.h
@@ -209,14 +209,14 @@ typedef struct _HV_PORT_INFO
 	struct
 	{
 	    HV_SYNIC_SINT_INDEX TargetSint;
-	    HV_VP_INDEX TargetVp;
+	    u32 TargetVp;
 	    u64 RsvdZ;
 	} MessagePortInfo;
 
 	struct
 	{
 	    HV_SYNIC_SINT_INDEX TargetSint;
-	    HV_VP_INDEX TargetVp;
+	    u32 TargetVp;
 	    u16 BaseFlagNumber;
 	    u16 FlagCount;
 	    u32 RsvdZ;
--- a/drivers/staging/hv/include/HvVpApi.h
+++ b/drivers/staging/hv/include/HvVpApi.h
@@ -20,32 +20,26 @@
  *   Hank Janssen  <hjanssen@microsoft.com>
  *
  */
+#ifndef __HVVPAPI_H
+#define __HVVPAPI_H
 
+/*
+ * The below CPUID leaves are present if VersionAndFeatures.HypervisorPresent
+ * is set by CPUID(HvCpuIdFunctionVersionAndFeatures).
+ */
+enum hv_cpuid_function {
+	HvCpuIdFunctionVersionAndFeatures		= 0x00000001,
+	HvCpuIdFunctionHvVendorAndMaxFunction		= 0x40000000,
+	HvCpuIdFunctionHvInterface			= 0x40000001,
+
+	/*
+	 * The remaining functions depend on the value of
+	 * HvCpuIdFunctionInterface
+	 */
+	HvCpuIdFunctionMsHvVersion			= 0x40000002,
+	HvCpuIdFunctionMsHvFeatures			= 0x40000003,
+	HvCpuIdFunctionMsHvEnlightenmentInformation	= 0x40000004,
+	HvCpuIdFunctionMsHvImplementationLimits		= 0x40000005,
+};
 
-#pragma once
-
-/* Virtual Processor Indices */
-
-typedef u32 HV_VP_INDEX, *PHV_VP_INDEX;
-
-
-/* The below CPUID leaves are present if VersionAndFeatures.HypervisorPresent */
-/* is set by CPUID(HvCpuIdFunctionVersionAndFeatures). */
-/* ========================================================================== */
-
-
-typedef enum _HV_CPUID_FUNCTION
-{
-    HvCpuIdFunctionVersionAndFeatures           = 0x00000001,
-    HvCpuIdFunctionHvVendorAndMaxFunction       = 0x40000000,
-    HvCpuIdFunctionHvInterface                  = 0x40000001,
-
-
-    /* The remaining functions depend on the value of HvCpuIdFunctionInterface */
-
-    HvCpuIdFunctionMsHvVersion                  = 0x40000002,
-    HvCpuIdFunctionMsHvFeatures                 = 0x40000003,
-    HvCpuIdFunctionMsHvEnlightenmentInformation = 0x40000004,
-    HvCpuIdFunctionMsHvImplementationLimits     = 0x40000005
-
-} HV_CPUID_FUNCTION, *PHV_CPUID_FUNCTION;
+#endif
